tests/TestCase.php

```diff
@@ -2,22 +2,10 @@
 
 namespace Spatie\LaravelPdf\Tests;
 
+use Orchestra\Testbench\Concerns\WithWorkbench;
 use Orchestra\Testbench\TestCase as Orchestra;
-use Spatie\LaravelPdf\PdfServiceProvider;
 
 class TestCase extends Orchestra
 {
-    protected function setUp(): void
-    {
-        parent::setUp();
-
-        $this->app->view->addLocation(__DIR__.'/TestSupport/Views');
```

Comment:
Now can be auto-discovered if we use `workbench/resources/views` and update `testbench.yaml`:

```yaml
discovers:
  views: true
```